scss/_patterns_contextual-menu.scss

```diff
@@ -63,11 +63,9 @@
     }
   }
 
-  .p-contextual-menu__toggle {
-    margin-bottom: $spv-inner--small + ($spv-nudge-compensation * 2);
```

Comment:
@lyubomir-popov I removed that, because I wasn't sure if it's necessary. This was changing the bottom margin on toggle buttons, so the value of `$input-margin-bottom` was not even correct.

So now the buttons have their default margin bottom, and we compensate it by moving contextual menu up (only if toggle is button).

Reply:
Well spotted, I can confirm the amount of spacing inserted after matches  that of buttons and inputs.